jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/CreateQueueDialog.kt

```diff
@@ -64,42 +66,44 @@ class CreateQueueDialog(
     }
 
     private fun validateFields(): ValidationInfo? {
-        if (queueName().isEmpty()) {
+        if (view.queueName.text.isEmpty()) {
             return ValidationInfo(message("sqs.create.validation.empty.queue.name"), view.queueName)
-        } else if (queueName().length > MAX_LENGTH_OF_QUEUE_NAME) {
-            return ValidationInfo(message("sqs.create.validation.long.queue.name", MAX_LENGTH_OF_QUEUE_NAME), view.queueName)
         }
-
-        if (view.fifoType.isSelected) {
-            if (!queueName().endsWith(".fifo")) {
-                return ValidationInfo(message("sqs.create.validation.fifo"), view.queueName)
-            } else if (!validateCharacters(queueName().substringBefore(".fifo"))) {
-                return ValidationInfo(message("sqs.create.validation.queue.name.invalid"), view.queueName)
-            }
-        } else {
-            if (!validateCharacters(queueName())) {
-                return ValidationInfo(message("sqs.create.validation.queue.name.invalid"), view.queueName)
+        if (!validateCharacters(view.queueName.text)) {
+            return ValidationInfo(message("sqs.create.validation.queue.name.invalid"), view.queueName)
+        }
+        if (queueName().length > MAX_LENGTH_OF_STANDARD_QUEUE_NAME) {
```

Comment:
This check is logically correct but weird, can you rename `MAX_LENGTH_OF_STANDARD_QUEUE_NAME` to `MAX_LENGTH_OF_QUEUE_NAME`